templates/managed/index.html

```diff
@@ -1,450 +1,592 @@
-            </p>
-        </div>
+  <div class="row u-sv3">
+    <div class="col-8">
+      <h2>Operations can be smarter</h2>
+      <p>Smart operations don’t consume your budgets. They help you and your team focus on the value creation, spending more time and effort achieving your business goals. In-house IT engineers are better positioned to prepare your IT for the future &ndash; let them do more driving and less firefighting.</p>
     </div>
+  </div>
+</section>
 
-    <div class="row u-vertically-spaced">
-        <div class="col-4 u-sv3">
-            <h3 class="p-heading--four">Business-centric IT</h3>
-            <p>You drive your business, we manage your cloud infrastructure and applications. Expand globally without hiring a global team.</p>
-        </div>
+<div class="u-fixed-width">
+    <hr>
+  </div>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      <h2>Freedom to choose</h2>
+      <p>Your cloud must be designed to scale from day one. It should allow your IT the flexibility to grow and give you freedom of choice. Run containers on top of OpenStack, integrated with your database and observability layer in a hybrid or a multi-cloud environment &ndash; all managed for you.</p>
+    </div>
+    <div class="col-6 u-align--center u-hide--small">
+      {{
+      image(
+      url="https://assets.ubuntu.com/v1/23cbb344-Design.png",
+      alt="Tailored Architecture",
+      width="347",
+      height="200",
+      hi_def=True,
+      ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<div class="u-fixed-width">
+  <hr>
+</div>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6 u-align--center u-hide--small">
+      {{ image(
+      url="https://assets.ubuntu.com/v1/50dea393-fixes+in+24hrs.svg",
+      alt="Operations",
+      width="200",
+      height="200",
+      hi_def=True,
+      ) | safe
+      }}
+    </div>
+    <div class="col-6">
+      <h2>Operations</h2>
+      <p>Operating your cloud services at scale is an important success factor. Experts in each component of your cloud solution are required 24x7 to keep it running. Hiring and training experts consumes a lot of effort, resources and budget. Our global team of technology specific experts all around the globe provides you with 24x7 reliable operations. </p>
+    </div>
+  </div>
+</section>
+
+<div class="u-fixed-width">
+  <hr>
+</div>
+
+<section class="p-strip">
+  <div class="row u-vertically-center">
+    <div class="col-6">
+      <h2>Predictable Costs</h2>
+      <p> You can’t improve what you can’t see, so we provide you with better visibility to give you more control over your cloud costs.</p>
+      <p> We build a robust cloud solution for you that provides dashboards for logging, monitoring and alerting. This provides you with the full visibility that enables predictive capacity planning to avoid surprises. </p>
```

Comment:
```suggestion
      <p> We build a robust cloud solution for you with all the dashboards for logging, monitoring and alerting to give full visibility and predictive capacity planning to avoid surprises.</p>
```